packages/data/src/components/with-select/test/index.js

```diff
@@ -64,6 +64,39 @@ describe( 'withSelect', () => {
 		} );
 	} );
 
+	it( 'passes through the component context to the registry store', () => {
+		const getSelectors = jest.fn();
+
+		const genericStore = {
+			getSelectors,
+			getActions: () => {},
+			subscribe: () => {},
+		};
+
+		registry.registerGenericStore( 'store1', genericStore );
+
+		function mapSelectToProps( select ) {
+			select( 'store1' );
+		}
+
+		const OriginalComponent = () => <div></div>;
+
+		const ComponentWithSelect = withSelect( mapSelectToProps )( OriginalComponent );
+
+		const testRenderer = TestRenderer.create(
+			<RegistryProvider value={ registry }>
+				<ComponentWithSelect />
+			</RegistryProvider>
+		);
+		const testInstance = testRenderer.root;
+
+		expect( getSelectors ).toHaveBeenCalledTimes( 1 );
+
+		const selectorContext = getSelectors.mock.calls[ 0 ][ 0 ];
```

Comment:
Just out of curiosity, are there any nicer test matchers/APIs for this? Not a big deal if no, but just curious 😄

Reply:
It doesn't appear so, this is from the Jest docs:

> // The first arg of the last call to the mock function was `42`
// (note that there is no sugar helper for this specific of an assertion)
expect(mockFunc.mock.calls[mockFunc.mock.calls.length - 1][0]).toBe(42);